packages/ui/src/utils/api.tsx

```diff
@@ -0,0 +1,143 @@
+import { XCircleIcon } from "@heroicons/react/24/outline";
+import * as keys from "../config/keys";
+import { useNotifications } from "../services/notifications";
+const apiURL = `${keys.CLIENT_HOME_PAGE_URL}/api/flags/`;
+
+export async function fetchGet(pathURL: string, config?: any) {
```

Comment:
I would like to keep the functional notation:

`const fetchGet = async () => .....`